.github/workflows/bump-golang.yml

```diff
@@ -18,13 +18,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        file: ['bump-golang.yml', 'bump-golang-7.17.yml']
+        file: ["bump-golang.yml", "bump-golang-7.17.yml"]
```

Comment:
what's the reason for this change?

Reply:
Autoformatting. There's a rule if a string is not interpolated, it should be single-quoted. IDK whether we follow that or not

Reply:
Reverted that